### DISCUSSION TITLE
Probably a bug in ClassificationReport

### DISCUSSION THREAD

**Discussion by LorenzoDag** [online-ml/river#1068] [Show and tell]
Hey everyone.

It seems that in ClassificationReport#__repr__ (line 127) .rstrip(".0") is used to format the Support value. This could be incorrect because rstrip removes all the occourrencies of "." and "0" until it encounters a different value.

Rstripping this way "4030.0" results in "403".

Replacing it with rstrip("0").rstrip(".") fix the bug.

I also tried casting the value to integer before casting in string and it works but the previous solution seems to be faster.

**Comment by MaxHalford** [comment]
Thanks a lot, just solved the issue in #1069.

**Reply by LorenzoDag** [reply]
Thank you for your work 👍🏼
